<?xml version="1.0" encoding="UTF-8"?><testrun name="SerializeTestCase" project="AuditMediator" tests="4" started="4" failures="1" errors="0" ignored="0">
  <testsuite name="org.codigolibre.auditbpmn.wso2mediator.SerializeTestCase" time="0.967">
    <testcase name="testXMLSerializeInsideVariable" classname="org.codigolibre.auditbpmn.wso2mediator.SerializeTestCase" time="0.766"/>
    <testcase name="testVariable" classname="org.codigolibre.auditbpmn.wso2mediator.SerializeTestCase" time="0.011"/>
    <testcase name="testLOGGER" classname="org.codigolibre.auditbpmn.wso2mediator.SerializeTestCase" time="0.003"/>
    <testcase name="testDetailLevels" classname="org.codigolibre.auditbpmn.wso2mediator.SerializeTestCase" time="0.187">
      <failure>
        <expected>{"businessProcessAudit":{"id":{"@nil":"true"},"name":{"@nil":"true"},"startTime":{"@nil":"true"},"endTime":{"@nil":"true"},"status":{"@nil":"true"}}}</expected>
        <actual>&lt;businessProcessAudit xmlns="urn:org:codigolibre:businessprocessaudit:type:v1.0.0"&gt;&lt;id&gt;SincronizacionAnuncios&lt;/id&gt;&lt;name&gt;Sincronizacion Anuncios OVESAE&lt;/name&gt;&lt;correlationID&gt;2124cf6b-470d-4e87-b226-2586b4c0cb26&lt;/correlationID&gt;&lt;startTime&gt;2017-03-03T09:46:00.024+01:00&lt;/startTime&gt;&lt;endTime&gt;2017-03-03T09:46:01.635+01:00&lt;/endTime&gt;&lt;status&gt;OK&lt;/status&gt;&lt;/businessProcessAudit&gt;</actual>junit.framework.ComparisonFailure: 
 Valid {"businessProcessAudit":{"id":{"@nil":"true"},"name":{"@nil":"true"},"startTime":{"@nil":"true"},"endTime":{"@nil":"true"},"status":{"@nil":"true"}}}
 Generated &lt;businessProcessAudit xmlns="urn:org:codigolibre:businessprocessaudit:type:v1.0.0"&gt;&lt;id&gt;SincronizacionAnuncios&lt;/id&gt;&lt;name&gt;Sincronizacion Anuncios OVESAE&lt;/name&gt;&lt;correlationID&gt;2124cf6b-470d-4e87-b226-2586b4c0cb26&lt;/correlationID&gt;&lt;startTime&gt;2017-03-03T09:46:00.024+01:00&lt;/startTime&gt;&lt;endTime&gt;2017-03-03T09:46:01.635+01:00&lt;/endTime&gt;&lt;status&gt;OK&lt;/status&gt;&lt;/businessProcessAudit&gt; expected:&lt;[{"businessProcessAudit":{"id":{"@nil":"true"},"name":{"@nil":"true"},"startTime":{"@nil":"true"},"endTime":{"@nil":"true"},"status":{"@nil":"true"}}}]&gt; but was:&lt;[&lt;businessProcessAudit xmlns="urn:org:codigolibre:businessprocessaudit:type:v1.0.0"&gt;&lt;id&gt;SincronizacionAnuncios&lt;/id&gt;&lt;name&gt;Sincronizacion Anuncios OVESAE&lt;/name&gt;&lt;correlationID&gt;2124cf6b-470d-4e87-b226-2586b4c0cb26&lt;/correlationID&gt;&lt;startTime&gt;2017-03-03T09:46:00.024+01:00&lt;/startTime&gt;&lt;endTime&gt;2017-03-03T09:46:01.635+01:00&lt;/endTime&gt;&lt;status&gt;OK&lt;/status&gt;&lt;/businessProcessAudit&gt;]&gt;
	at junit.framework.Assert.assertEquals(Assert.java:85)
	at org.codigolibre.auditbpmn.wso2mediator.SerializeTestCase.testDetailLevels(SerializeTestCase.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
  </testsuite>
</testrun>
